### PR TITLE
Disable tide validation while we explore CODEOWNERS

### PR DESCRIPTION
**Pull request by mattmoor**

### CODE REVIEW

**Comment by mattmoor** [comment]
cc @chaodaiG @chizhg @cjwagner

File: prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml [hunk 1]
```diff
@@ -84,6 +84,9 @@ presubmits:
             - --job-config-path=../../knative/test-infra/config/prod/prow/jobs
             - --plugin-config=prow/knative/plugins.yaml
             - --strict
+            # TODO: Re-enable these once we're done experimenting with CODEOWNERS in net-contour.
+            - --exclude-warning=mismatched-tide
```
Comment (cjwagner) [inline prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml:88]: This is already excluded on line 92.

**Review by cjwagner — CHANGES_REQUESTED** [review]
For reference here is the thread explaining the motivation for this: https://knative.slack.com/archives/CCSNR4FCH/p1613067660097200

File: prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml (hunk 1 above)
Comment (mattmoor) [reply prow/prowjobs/GoogleCloudPlatform/oss-test-infra/gcp-oss-test-infra-config.yaml]: I removed this and moved the whole block down.

**Review by cjwagner — APPROVED** [review]

**Pull request merged**